packages/web-twig/tests/components-fixtures/fileUploaderAttachment.twig

```diff
@@ -1,8 +1,17 @@
 <FileUploaderAttachment />
 
+<!-- Render with image preview -->
+<FileUploaderAttachment hasImagePreview />
```

Comment:
Could you maybe add more test cases for `hasImagePreview` and `imagePreview` scenarios? I think we need to test:

1. template, don't generate preview,
2. template, do generate preview,
3. attachment list, show an icon (possible duplicate of the no-props test case),
4. attachment list, image is provided by backend (possible duplicate of or overlap with the all-props test case).